src/JsonSerializer/JsonSerializer.php

```diff
@@ -49,11 +49,9 @@ class JsonSerializer
     protected $objectMappingIndex = 0;
 
     /**
-     * Closure serializer instance
-     *
-     * @var ClosureSerializerInterface
+     * Closure manager
```

Comment:
@jrbasso Shouldn't this be typed to `ClosureSerializerManager`?

Reply:
@cjsaylor Just fixed it.